### PR TITLE
usm: Added documentation for bare metal support

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

Added documentation for USM on bare metal machine 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/universal_service_monitoring/_index.md [hunk 1]
```diff
@@ -821,6 +821,34 @@ DD_SYSTEM_PROBE_NETWORK_HTTP_REPLACE_RULES=[{"pattern":"<drop regex>","repl":""}
 
 {{< /tabs >}}
 
+### Non containerized support
```
Comment (estherk15) [inline content/en/universal_service_monitoring/_index.md:824]:
```suggestion
### Non-containerized support
```

File: content/en/universal_service_monitoring/_index.md [hunk 2]
```diff
@@ -821,6 +821,34 @@ DD_SYSTEM_PROBE_NETWORK_HTTP_REPLACE_RULES=[{"pattern":"<drop regex>","repl":""}
...
+
+Universal Service Monitoring available in *beta* to monitor services running bare-metal on Linux virtual machines.
```
Comment (estherk15) [inline content/en/universal_service_monitoring/_index.md:826]:
```suggestion
<div class="alert alert-info">
Universal Service Monitoring is available in *beta* to monitor services running bare-metal on Linux virtual machines.
</div>
```

**Review by estherk15 — APPROVED** [review]
Looks good, left some suggestions!

File: content/en/universal_service_monitoring/_index.md (hunk 1 above)
Comment (sunhay) [reply content/en/universal_service_monitoring/_index.md]: I'd suggest we change the title to: `Support for non-containerized services (Public Beta)`

**Comment by sunhay** [comment]
Lets make sure we update the following sections as well, and if possible -- link directly to the beta instructions

- Setup > Supported versions and compatibility
- Setup > Prerequisites

File: content/en/universal_service_monitoring/_index.md (hunk 1 above)
Comment (sunhay) [reply content/en/universal_service_monitoring/_index.md]: Lets also move this section to be immediately after **Exploring your services**

**Comment by sunhay** [comment]
Few comments on my side

File: content/en/universal_service_monitoring/_index.md (hunk 1 above)
Comment (guyarb) [reply content/en/universal_service_monitoring/_index.md]: @sunhay, the linter does not allow having Public Beta in the headline (capitalization issues)
is it enough to have the "public beta" message here?
image

File: content/en/universal_service_monitoring/_index.md [hunk 3]
```diff
@@ -36,8 +36,9 @@ Universal Service Monitoring (USM) provides visibility into your service health
 
 Required Agent version
 : Universal Service Monitoring requires that the Datadog Agent installed alongside your containerized service be at least version 6.40 or 7.40.
+For non-containerized services see instructions [here](7)
```
Comment (sunhay) [inline content/en/universal_service_monitoring/_index.md:39]: I think you meant to put this below under Prerequisites :)

File: content/en/universal_service_monitoring/_index.md (hunk 1 above)
Comment (sunhay) [reply content/en/universal_service_monitoring/_index.md]: I think that should be fine! @estherk15 any thoughts?

File: content/en/universal_service_monitoring/_index.md (hunk 3 above)
Comment (guyarb) [reply content/en/universal_service_monitoring/_index.md]: great catch
moved it

File: content/en/universal_service_monitoring/_index.md [hunk 4]
```diff
@@ -56,6 +56,7 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 
 - If on Linux:
     - Your service is running in a container.
+    - For non-containerized services see instructions [here][7]
```
Comment (estherk15) [inline content/en/universal_service_monitoring/_index.md:59]: This is the format we use for anchor links (though your way also works in my local build, not sure how it is translated in the build)
```suggestion
    - For non-containerized services see the [instructions here](#support-for-non-containerized-services).
```

File: content/en/universal_service_monitoring/_index.md [hunk 5]
```diff
@@ -831,3 +861,4 @@ DD_SYSTEM_PROBE_NETWORK_HTTP_REPLACE_RULES=[{"pattern":"<drop regex>","repl":""}
 [4]: /monitors/types/apm/?tab=apmmetrics
 [5]: /dashboards/
 [6]: /service_management/service_level_objectives/metric/
+[7]: #support-for-non-containerized-services
```
Comment (estherk15) [inline content/en/universal_service_monitoring/_index.md:864]:
```suggestion
```

**Review by estherk15 — APPROVED** [review]

File: content/en/universal_service_monitoring/_index.md (hunk 1 above)
Comment (estherk15) [reply content/en/universal_service_monitoring/_index.md]: Yes! This looks great!

**Pull request merged**